sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample_AnalyzeOperationAsync_ManualPolling.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.AI.TextAnalytics.Models;
+using Azure.AI.TextAnalytics.Tests;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.TextAnalytics.Samples
+{
+    [LiveOnly]
+    public partial class TextAnalyticsSamples: SamplesBase<TextAnalyticsTestEnvironment>
+    {
+        [Test]
+        public async Task AnalyzeOperationAsync_ManualPolling()
+        {
+            string endpoint = TestEnvironment.Endpoint;
+            string apiKey = TestEnvironment.ApiKey;
+
+            var client = new TextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+
+            #region Snippet:AnalyzeOperationBatchAsyncManualPolling
+            string document = @"We went to Contoso Steakhouse located at midtown NYC last week for a dinner party, 
+                                and we adore the spot! They provide marvelous food and they have a great menu. The
+                                chief cook happens to be the owner (I think his name is John Doe) and he is super 
+                                nice, coming out of the kitchen and greeted us all. We enjoyed very much dining in 
+                                the place! The Sirloin steak I ordered was tender and juicy, and the place was impeccably
+                                clean. You can even pre-order from their online menu at www.contososteakhouse.com, 
+                                call 312-555-0176 or send email to order@contososteakhouse.com! The only complaint 
+                                I have is the food didn't come fast enough. Overall I highly recommend it!";
+
+            var batchDocuments = new List<TextDocumentInput>
+            {
+                new TextDocumentInput("1", document)
+                {
+                     Language = "en",
+                }
+            };
+
+            AnalyzeOperationOptions operationOptions = new AnalyzeOperationOptions()
+            {
+                KeyPhrasesTaskParameters = new KeyPhrasesTaskParameters(),
+                EntitiesTaskParameters = new EntitiesTaskParameters(),
+                PiiTaskParameters = new PiiTaskParameters(),
+                DisplayName = "AnalyzeOperationSample"
+            };
+
+            AnalyzeOperation operation = await client.StartAnalyzeOperationBatchAsync(batchDocuments, operationOptions);
+
+            TimeSpan pollingInterval = new TimeSpan(1000);
+
+            while (true)
+            {
+                await operation.UpdateStatusAsync().ConfigureAwait(false);
```

Comment:
could we delete the `.ConfigureAwait(false);` from both samples? so far we don't use it in any other samples and I will like to keep consistency on it.
For context, the usage of it depends more on the user application, and I wouldn't want them to think that they will always have to use it.